mimic/test/test_maas.py

```diff
@@ -270,6 +290,24 @@ def test_update_alarm(self):
         alarm2 = self.get_responsebody(resp)['values'][0]['alarms'][0]
         self.assertEquals(alarm['label'], alarm2['label'])
 
+    def test_partial_update_alarm(self):
+        """
+        Update an alarm, fields not specified in the body don't change.
```

Comment:
Minor nitpick here - it's generally better to phrase test assertions positively.  So rather than "fields not … don't …", instead, "When a request is received that updates an alarm, fields not specified in that request's body remain the same."